source/extensions/filters/http/aws_request_signing/aws_request_signing_filter.h

```diff
@@ -17,9 +17,11 @@ namespace AwsRequestSigningFilter {
  * All stats for the AWS request signing filter. @see stats_macros.h
  */
 // clang-format off
-#define ALL_AWS_REQUEST_SIGNING_FILTER_STATS(COUNTER)                                                           \
-  COUNTER(signing_added)                                                                        \
-  COUNTER(signing_failed)
+#define ALL_AWS_REQUEST_SIGNING_FILTER_STATS(COUNTER)                                              \
+  COUNTER(signing_added)                                                                           \
+  COUNTER(signing_failed)                                                                          \
+  COUNTER(payload_signing_added)                                                                   \
```

Comment:
You'll want to add docs for these new stats over here:
https://github.com/envoyproxy/envoy/blob/1d1b708c7bf6efa02c41d9ce22cbf1e4a1aeec2c/docs/root/configuration/http/http_filters/aws_request_signing_filter.rst#L43